### PR TITLE
implement freezero(), use localtime_s() instead of localtime()

### PR DESCRIPTION
**Pull request by bagajjal**
https://github.com/PowerShell/Win32-OpenSSH/issues/1121

### CODE REVIEW

**Comment by manojampalam** [comment]
Add declaration in the relevant header, update config.h.vs to indicate we have its implementation.

**Review by bingbing8 — APPROVED** [review]

File: contrib/win32/win32compat/misc.c [hunk 1]
```diff
@@ -1285,42 +1285,42 @@ getusergroups(const char *user, int *ngroups)
...
-				current_group = (*ngroups)++;
-				swprintf_s(formatted_group, ARRAYSIZE(formatted_group), L"%s", name);
-				_wcslwr_s(formatted_group, ARRAYSIZE(formatted_group));
-				debug3("Added group '%ls' for user '%s'.", formatted_group, user);
-				user_groups[current_group] = utf16_to_utf8(formatted_group);
-				if (user_groups[current_group] == NULL) {
-					errno = ENOMEM;
-					goto cleanup;
-				}
+			/* lookup the account name for this sid */
```
Comment (manojampalam) [inline contrib/win32/win32compat/misc.c:1288]: Why is this showing up as a change?

Comment (bagajjal) [reply contrib/win32/win32compat/misc.c:1288]: The checkin file from latestw_all has CRLF issue..

**Comment by ivanov-i** [comment]
What about openbsd-compat\freezero.c? Seems weird to have two identical implementations.

**Comment by bagajjal** [comment]
@ivanov-i - We don't want to take dependency on OpenBSD-compact.lib. This gives us more flexibility if we need to diverge and have a windows specific code if required.

**Pull request merged**